src/main/java/de/caritas/cob/statisticsservice/api/statistics/model/statisticsevent/meta/DeleteAccountMetaData.java

```diff
@@ -0,0 +1,12 @@
+package de.caritas.cob.statisticsservice.api.statistics.model.statisticsevent.meta;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+@Data
+public class DeleteAccountMetaData {
+
```

Comment:
isn't the userID missing here?

Reply:
no it's not, because it's part of the aggregating object DeleteAccountStatisticsEventMessage, so no need to put same information in the metadata